controllers/platform/services/services.go

```diff
@@ -0,0 +1,298 @@
+	c := containerSpec.DeepCopy()
+	err := mergo.Merge(c, j.platform.Spec.Services.JobService.PodTemplate.Container.ToContainer(), mergo.WithOverride)
+	return c, err
+}
+
+func (j JobService) ConfigurePersistence(containerSpec *corev1.Container) *corev1.Container {
+
+	if j.platform.Spec.Services.JobService.Persistence != nil && j.platform.Spec.Services.JobService.Persistence.PostgreSql != nil {
+		c := containerSpec.DeepCopy()
+		c.Image = j.GetServiceImageName(constants.PersistenceTypePostgreSQL)
+		c.Env = append(c.Env, configurePostgreSqlEnv(j.platform.Spec.Services.JobService.Persistence.PostgreSql, j.GetServiceName(), j.platform.Namespace)...)
+		return c
+	}
+	return containerSpec
+}
+
+func (j JobService) MergePodSpec(podSpec corev1.PodSpec) (corev1.PodSpec, error) {
+	c := podSpec.DeepCopy()
+	err := mergo.Merge(c, j.platform.Spec.Services.JobService.PodTemplate.PodSpec.ToPodSpec(), mergo.WithOverride)
+	return *c, err
+}
+
+func configurePostgreSqlEnv(postgresql *operatorapi.PersistencePostgreSql, databaseSchema, databaseNamespace string) []corev1.EnvVar {
+	if len(postgresql.ServiceRef.DatabaseSchema) > 0 {
+		databaseSchema = postgresql.ServiceRef.DatabaseSchema
+	}
+	if len(postgresql.ServiceRef.Namespace) > 0 {
+		databaseNamespace = postgresql.ServiceRef.Namespace
+	}
+	dataSourcePort := 5432
+	if postgresql.ServiceRef.Port != nil {
+		dataSourcePort = *postgresql.ServiceRef.Port
+	}
+	databaseName := "sonataflow"
+	if len(postgresql.ServiceRef.DatabaseName) > 0 {
+		databaseName = postgresql.ServiceRef.DatabaseName
+	}
+	dataSourceUrl := "jdbc:" + constants.PersistenceTypePostgreSQL + "://" + postgresql.ServiceRef.Name + "." + databaseNamespace + ":" + strconv.Itoa(dataSourcePort) + "/" + databaseName + "?currentSchema=" + databaseSchema
+	if len(postgresql.JdbcUrl) > 0 {
+		dataSourceUrl = postgresql.JdbcUrl
+	}
+	secretRef := corev1.LocalObjectReference{
+		Name: postgresql.SecretRef.Name,
+	}
+	quarkusDatasourceUsername := "POSTGRESQL_USER"
+	if len(postgresql.SecretRef.UserKey) > 0 {
+		quarkusDatasourceUsername = postgresql.SecretRef.UserKey
+	}
+	quarkusDatasourcePassword := "POSTGRESQL_PASSWORD"
+	if len(postgresql.SecretRef.PasswordKey) > 0 {
+		quarkusDatasourcePassword = postgresql.SecretRef.PasswordKey
+	}
+	return []corev1.EnvVar{
+		{
+			Name: "QUARKUS_DATASOURCE_USERNAME",
+			ValueFrom: &corev1.EnvVarSource{
+				SecretKeyRef: &corev1.SecretKeySelector{
+					Key:                  quarkusDatasourceUsername,
+					LocalObjectReference: secretRef,
+				},
+			},
+		},
+		{
+			Name: "QUARKUS_DATASOURCE_PASSWORD",
+			ValueFrom: &corev1.EnvVarSource{
+				SecretKeyRef: &corev1.SecretKeySelector{
+					Key:                  quarkusDatasourcePassword,
+					LocalObjectReference: secretRef,
+				},
+			},
+		},
+		{
+			Name:  "QUARKUS_DATASOURCE_DB_KIND",
+			Value: constants.PersistenceTypePostgreSQL,
+		},
+		{
+			Name:  "QUARKUS_HIBERNATE_ORM_DATABASE_GENERATION",
```

Comment:
This env var don't apply for the jobs-service

Reply:
ack

Reply:
Done.